lib/flipper-file/file-tool.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+#include <stdint.h>
+#include <mlib/m-string.h>
+#include <storage/storage.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+extern const char flipper_file_eoln;
+extern const char flipper_file_eolr;
+
+/**
+ * Negative seek helper
+ * @param file 
+ * @param offset 
+ * @return bool 
+ */
+bool file_tool_seek(File* file, int32_t offset);
```

Comment:
File tool is quite strange name, file helper is more common and suitable

Reply:
Fixed in 0a72f6e53cfd9b6cce00573fb7805bd9534edd89